docs/UserGuide.md

```diff
@@ -129,14 +127,32 @@ More examples:
 
 ### Importing your timetable : `import`
 
-Imports your NUSMods timetable data into your schedule.
-
 Format: `import url/YOUR_NUSMODS_URL`
 
-* Lesson data based on your NUSMods timetable will be added into your schedule.
-* `YOUR_NUSMODS_URL` is obtained by clicking on the "Share/Sync" timetable icon at the NUSMods website.
+You can import your NUSMods timetable data into ProductiveNUS by providing the URL to your NUSMods timetable share
+link. Imported lesson information can be found in the `Upcoming tasks` list as shown.
+ 
+DIAGRAM
+ 
+Follow these steps and try importing your timetable:
 
-Examples:
+1) At your NUSMods timetable website, click on the "Share/Sync" button to obtain your timetable share link URL.
+
+DIAGRAM
+
+2) The URL obtained will be `YOUR_NUSMODS_URL` to be used in the import command.
+
+3) Added lesson information can be found in the task list.
+
```

Comment:
It may be good to phrase it as "`your` task list" instead of "`the` task list" to make it sound more user-centric, though this is just a minor nitpick.

Reply:
Amended